config/zones.json

```diff
@@ -3922,18 +3922,18 @@
       [-5.705947224999932, 42.65362965999999]
     ],
     "capacity": {
-      "biomass": 876,
+      "biomass": 700,
       "coal": 0,
-      "gas": 4580,
+      "gas": 4520,
       "geothermal": 0,
-      "hydro": 4372,
-      "hydro storage": 2827,
+      "hydro": 4578,
+      "hydro storage": 3585,
       "nuclear": 0,
       "oil": 0,
-      "solar": 1364,
-      "wind": 5528
+      "solar": 1616,
+      "wind": 5389
     },
-    "contributors": ["corradio", "nessie2013", "Falconet8"],
```

Comment:
Why did you remove Falconet8 as a contributor?

Reply:
Falconet8 is me. I changed the username.
What did you mean about the source?

Reply:
I see what you meant about the source. I pressed commit.

Reply:
Looks like something failed regarding the review checks. I don't understand any of it so I'll leave to someone else.